operator/images/access-setup/content/bin/setup_work_dir.sh

```diff
@@ -49,7 +54,7 @@ Example:
 }
 
 parse_args() {
-  KUSTOMIZATION=${KUSTOMIZATION:-github.com/openshift-pipelines/pipeline-service/operator/gitops/argocd?ref=main}
+  KUSTOMIZATION=${KUSTOMIZATION:-github.com/openshift-pipelines/pipeline-service/operator/gitops/argocd/overlays/dev?ref=main}
```

Comment:
That change should not be in this script, but made by using the right parameter when calling the script from developer/openshift/dev_setup.sh.